src/visitors/semantic/completion/results.rs

```diff
@@ -0,0 +1,240 @@
+use crate::protocol::responses::{
+    CompletionItem, CompletionItemKind, InsertTextFormat,
+};
+use crate::shared::signatures::get_argument_names;
```

Comment:
is there anything new here? I'm having difficulty finding diffs between the old and new 😬

Reply:
nothing new, just breaking up some large files into their own modules

Reply:
sweet, thanks for the heads up 👍